### PR TITLE
Add a news post for RA:GL S2 registrations, and an openra development update

### PR DESCRIPTION
**Pull request by obrakmann**

### CODE REVIEW

**Comment by pchote** [comment]
Looks good to me 👍

File: content/news/ragls2-registrations.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+<hr/>
+###OpenRA development news
+
+While the next release of OpenRA is not quite ready yet, we would like to give you a sneak peek at what we have been up to in the last few months and some of the things that you can expect to see in the hopefully not too distant future.
+
+<div style="text-align:center" markdown="1">
+![Global chat lobby](/images/news/20160821-sneakpeek-ai-capturing.png)
+</div>
+
+The AI will gain the ability to capture buildings.
```
Comment (pchote) [inline content/news/ragls2-registrations.md]: You could refer to the other AI fixes too.

Comment (obrakmann) [inline content/news/ragls2-registrations.md]: done

**Comment by Mailaender** [comment]
Good idea :+1: leaving a sign of life is important.

**Comment by Micr0Bit** [comment]
thx @obrakmann :+1:

**Comment by SoScared** [comment]
Thanks! There's gonna be hell of a lot of Recruits this season thanks to advertisements such as this.

**Comment by pchote** [comment]
Do we really want to reuse the same red alert 1 logo for the GL after all the panty-twisting angst the season 1 advertisements caused?

**Comment by obrakmann** [comment]
Wasn't the issue with the background image? Or was that something different again?

I remember people complaining that it was advertised as a red alert league "when it isn't really red alert", but I really don't feel like catering to those people that feel offended by that.

**Comment by SoScared** [comment]
Meh, it mostly triggered some ortodox CnCnet forum staffers tho. There's a spottable OpenRA "watermark" on it on Season 2.

ragl-s02_full1080

**Comment by Mailaender** [comment]
As this is season 2, it makes sense to use recognizable imagery compared to https://github.com/OpenRA/OpenRAWeb/pull/304. When @ElectronicArts doesn't want us to use the Command & Conquer trademark and logo, we can replace it at any time with something neutral.

**Pull request merged**